.github/workflows/gurobi-auto-update.yml

```diff
@@ -0,0 +1,91 @@
+name: Gurobi Auto Updater
+
+on:
+  schedule:
+    - cron: '15 1 * * 3' # Every Wednesday at 1:15 AM UTC
+  workflow_dispatch:
+
+jobs:
+  update-gurobi-version:
+    if: github.ref == 'refs/heads/main'
```

Comment:
I think you should be able to enforce that this only runs on the main branch by configuring the branch in the `on:` settings above.

Reply:
Would be nice, yes. I cannot figure out how to include this in schedule though, so I would keep this line